hq/app/views/iam/iamCredReportBody.scala.html

```diff
@@ -29,13 +29,12 @@
                     <td>@views.html.fragments.allKeyStatus(cred.key1Status, cred.key2Status)</td>
                     <td>@views.html.fragments.mfa(cred.hasMFA)</td>
                     <td>
-                        <span>
-                            @ReportDisplay.toDayString(cred.lastActivityDay)
-                        </span>
+                        <span>@ReportDisplay.toDayString(cred.lastActivityDay)</span>
```

Comment:
Doing this for consistency with the other template. I think this table and the use of fragments could be refactored a bit, and I would like to allow machineUsers and humanUsers to sort together.

Targets for a future PR! 😁